### DISCUSSION TITLE
Battery level bar

### DISCUSSION THREAD

**Discussion by cfkmo** [Clooos/Bubble-Card#722] [Questions about config, custom styles and templates] [answered]
Hej.

Is it possible to use the battery as header entity in a PopUp Card and to show the actual level like a (partly) filled bar?! 

Like it is done in the header for lights
![Screenshot 2024-08-19 120034](https://github.com/user-attachments/assets/86afa541-6c78-426a-b998-68d2e811c811)

like this one:
![ed0b8a6b094040f345acb57fadbabf802160c137](https://github.com/user-attachments/assets/e9776d18-0787-4440-83e1-135a9a20ffff)

regards, Christian

**Comment by koleos6** [comment]
Hello, I agree a new type of card that look like the slider one but for a status (battery or other %). with the custom styles we also could change the color depending of the %

**Comment by pdwonline** [comment]
seems a read-only variant of the slider would do..

**Comment by MrBearPresident** [comment, marked as answer]
See #928
